modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -177,6 +177,20 @@ describe('TSS Ecdsa Utils:', async function () {
       should.not.exist(backupKeyShares.bitGoHeldKeyShares);
     });
 
+    it('should get the correct bitgo gpg key based on coin and feature flags', async function() {
+      const nitroGPGKeypair = await openpgp.generateKey({
+        userIDs: [
+          {
+            name: 'bitgo nitro',
+            email: 'bitgo@test.com',
+          },
+        ],
+      });
+      const nockGPGKey = await nockGetBitgoPublicKeyBasedOnFeatureFlags(coinName, 'enterprise', nitroGPGKeypair);
+      const bitgoGpgPublicKey = await tssUtils.getBitgoGpgPubkeyBasedOnFeatureFlags('enterprise');
```

Comment:
'enterprise' value here is confusing since it looks like a magic value. It would be more readable if it looked more like an enterprise id.

Reply:
Updated it to `enterprise_id`